src/scripts/main.js

```diff
@@ -1,3 +1,203 @@
 'use strict';
 
-// write your code here
+const startBtn = document.querySelector('.start');
+const score = document.querySelector('.game-score');
+const startMsg = document.querySelector('.message-start');
+const winMsg = document.querySelector('.message-win');
+const loseMsg = document.querySelector('.message-lose');
+
+const fieldRows = [...document.querySelectorAll('.field-row')];
+const rows = [];
+
+for (const row of fieldRows) {
+  rows.push([...row.children]);
+}
+
+const columns = Array.from({ length: rows.length }, () => ([]));
+
+for (const row of rows) {
+  for (let i = 0; i < row.length; i++) {
+    columns[i].push(row[i]);
+  }
+}
+
+let cells = [];
+
+for (const row of rows) {
+  cells = cells.concat(row);
+}
+
```

Comment:
There is method like `arr.flat()` 

```
const arr1 = [0, 1, 2, [3, 4]];

console.log(arr1.flat());
// Expected output: Array [0, 1, 2, 3, 4]
```

You can use it as well `const cells = rows.flat()`

Reply:
Nice method, thank you)